reviews/2020-flux-annual.md

```diff
@@ -0,0 +1,145 @@
+for Flux. Contributions have been growing fairly steadily over the
+year ([devstats
+chart](https://flux.devstats.cncf.io/d/7/companies-contributing-in-repository-groups?orgId=1)).
+
+It's also worth looking at the [commits by
+repository](https://flux.devstats.cncf.io/d/2/commits-repository-groups?orgId=1&from=now-1y&to=now&var-period=w&var-repogroups=All)
+chart, which shows a change in emphasis. In words --
+
+ * Many commits before landing in Sandbox 2019-08-15 involved
+   decoupling from Weave Cloud, splitting out the Helm Operator, etc.
+ * In the months before and after CNCF Sandbox we had added features
+   and integrations like kustomize, support for encrypted manifests
+   with SOPS and lots of other improvements, so in the months after
+   that we focused on stabilisation, consolidation, docs, etc.
+ * Contributions largely stayed in the same ball-park, with a growing
+   proportion of contributions in fluxcd/helm-operator.
+ * In April 2020 we started experimentation around a more distributed
+   and focused delivery model within the Flux project, called the
+   GitOps Toolkit. It uses targeted controllers, for e.g. handling of
+   git sources, updating images, kustomization and more. This will
+   help us land some long-standing feature requests and be the
+   foundation of Flux version 2.
+
+### Maintainers
+
+Our current maintainers are listed in
+[fluxcd/flux](https://github.com/fluxcd/flux/blob/master/MAINTAINERS)
+and
+[fluxcd/helm-operator](https://github.com/fluxcd/helm-operator/blob/master/MAINTAINERS)
+-- three out of four working for Weaveworks.
+
+Some of the components of the GitOps Toolkit are seeing bigger and
+more steady contributions from people outside Weaveworks and we can
+hope to bring some people on board as maintainers of these projects.
+
+### Adoption
+
+We keep a self-selecting list of production users. This was the list
+at the time Flux became a CNCF project:
+
+    https://github.com/fluxcd/flux/tree/1.14.0#who-is-using-flux-in-production
+
+.. and this is the list today:
+
+    https://github.com/fluxcd/flux/blob/master/README.md#who-is-using-flux-in-production
+
+(i.e., it has almost tripled). Flux was in the ADOPT sector of the
+[CNCF Technology Radar for Continuous
+Delivery](https://www.cncf.io/blog/2020/06/12/introducing-the-cncf-technology-radar/),
+meaning it was widely adopted and recommended among the CNCF end user
+community.
+
+There are some proxy metrics for adoption:
+
+ * the #flux channel in CNCF Slack now has >1100 users;
+ * GitHub stars have gone from about 2500 to >5300 in the last year;
+ * we believe (based on optional telemetry in fluxd) that the install
+   base is on the order of tens of thousands of instances.
+
+### Performance against goals
+
+This is the Flux project's first review, so there are no goals already
+on record. However, there are some measures of progress:
+
+ * We have been talking to users and customers a lot. The Flux
```

Comment:
Developer advocacy for GitOps in the community is the great effort. Thank you for doing that!